src/components/Planetarium/Planetarium.tsx

```diff
@@ -8,23 +8,26 @@ const Planetarium: React.FC<AllData> = ({ allPlanets, sortKey }): JSX.Element =>
   const addDescriptor = (data: number, key: string) => {
     let descriptor: string = '';
     if (key === 'distance_from_sun' || key === 'diameter') {
-      descriptor = 'kilometers';
+      descriptor = 'km';
     } else if (key === 'mass') {
-      descriptor = 'kilograms';
+      descriptor = 'quintillion kg';
     } else if (key === 'length_of_day') {
       descriptor = 'hours';
     } else if (key === 'length_of_year') {
       descriptor = 'days'
     } else if (key === 'number_of_moons') {
       descriptor = 'moons'
     } else if (key === 'gravity') {
-      descriptor = 'metres per second squared'
+      descriptor = 'm/s²'
     }
     return (`${data} ${descriptor}`)
   }
 
   const planetCards = allPlanets.map(planet => {
-    const factData: string | number = planet[sortKey as keyof PlanetBio]
+    let factData: string | number = planet[sortKey as keyof PlanetBio];
+    if (!factData) {
```

Comment:
This is my bad, but this where we are losing moons! 
This should be `if (factData === undefined)`  and that seems to work